packages/tools/lib/generate-json-imports/illustrations.js

```diff
@@ -0,0 +1,75 @@
+const fs = require("fs").promises;
```

Comment:
When I build the project I get a loader for AllIllustrations.js by mistake, please have a look:

<img width="627" alt="Screenshot 2022-10-24 at 12 24 03" src="https://user-images.githubusercontent.com/15702139/197493794-bc26ecc4-8c13-47aa-9905-858a5c05b09c.png">